tests/db_modes_test.sh

```diff
@@ -47,7 +47,8 @@ run_expect_success() {
    local NODEOS_PID=$!
    sleep 10
    kill $NODEOS_PID
-   wait $NODEOS_PID
+   rc=0
+   wait $NODEOS_PID && rc=$? || rc=$?
```

Comment:
Once retrieved, test for macOS and explicitly return 0 if rc is 0 or 127.  If not macOS, just return rc.

Reply:
Hi Jonathan, actually ubuntu also return 127, if you try wait 98765, and the echo "$?"
jonathan.giszczak-ext:speech_balloon:  3:55 PM
Ah.  In that case, no need for the platform check.  Just check for 0 and 127 and return zero, otherwise return rc.